src/wp-admin/js/tags-box.js

```diff
@@ -95,6 +175,19 @@ var tagBox, array_unique_noempty;
 						'<span class="screen-reader-text">' + window.tagsSuggestL10n.removeTerm + ' ' + val + '</span>' +
 						'</button>' );
 
+					/**
+					 * Handles the click and keypress event when an event is called on a tag and reparses the tags
+					 * in the tag box.
+					 * Also sets the focus on the new tag field is the enter key is pressed.
+					 *
+					 * @summary Handles the click and keypress event when an event is called on a tag.
```

Comment:
`when an event` is the same event as the click/keypress event. Now it looks like any random event is triggering something.